torch/_dynamo/compiled_autograd.py

```diff
@@ -65,6 +66,40 @@ def maybe_clone(x):
     return x
 
 
+# We lazily bind "functional backward" variants for PyTorch built-in autograd
+# nodes to this class. Example: torch._dynamo.compiled_autograd.ops.MulBackward0
+# Each "functional backward" is bound the first time the node's apply_with_saved
+# function is called. It's possible to avoid lazy binding and instead bind
+# all of this upfront (perhaps at import time) via codegen changes.
+class OpNamespace:
+    def add(self, name, fn):
+        assert not hasattr(self, name)
+        result = Op(name, fn)
+        torch._dynamo.allow_in_graph(result)
+        setattr(self, name, result)
+        return result
+
+    def get(self, name):
+        assert hasattr(self, name)
```

Comment:
Is this line needed?  I'd expect the next line to throw n AttributeError exception if this is false.